### PR TITLE
Mitigate unsafe math in the claim_vesting

### PR DESCRIPTION
**Pull request by simonsso**
SR audit 18 

solves https://github.com/Polkadex-Substrate/audit-sr-labs/issues/18

Still no unit test for this function.

### CODE REVIEW

File: pallets/fungible-assets/src/lib.rs [hunk 1]
```diff
@@ -241,14 +239,15 @@ decl_module! {
         pub fn claim_vesting(origin, identifier: T::Hash, asset_id: T::CurrencyId) -> DispatchResult {
             let who: T::AccountId = ensure_signed(origin)?;
             let current_block_no = <system::Pallet<T>>::block_number();
-
+            assert_eq!(1,0,"code in use");
```
Comment (mambisi) [inline pallets/fungible-assets/src/lib.rs:242]: I don't know the purpose of this

**Review by mambisi — COMMENTED** [review]
Please check the comments

File: pallets/fungible-assets/src/lib.rs (hunk 1 above)
Comment (simonsso) [reply pallets/fungible-assets/src/lib.rs:242]: This should not be there!!! Good spoted!

**Review by mambisi — APPROVED** [review]
LGTM

**Pull request merged**